pkg/swap/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.5
+
+# swap command - only minimal Alpine install
+
+# use util-linux to get swapfile utils
+#   at some point, hopefully use our own mkswap and swapon
+RUN apk add --update util-linux
+
+# add the entrypoint and make it executable
+COPY . ./
+RUN chmod +x swap.sh
+RUN ln -s swap.sh swap
```

Comment:
Yeah, but the hashing algorithm you used depends on having everything in the container. I like `swap.sh` because of vim and Atom and other editors, but `swap` as a command is cleaner. :-)